guider.h

```diff
@@ -266,12 +267,15 @@ class Guider : public wxWindow
 
     void Reset(bool fullReset);
     void EnableMeasurementMode(bool enabled);
-    void SetMinStarHFD(double val);
     double GetMinStarHFD() const;
     double GetMinStarHFDFloor() const;
     double GetMinStarHFDDefault() const;
-    void SetMinStarSNR(double val);
-    double getMinStarSNR() const;
+    void SetMinStarHFD(double val);
+    double GetMaxStarHFD() const;
+    void SetMaxStarHFD(double val);
+    void SetAFMinStarSNR(double val);
+    double GetAFMinStarSNR() const;

```

Comment:
curious why these methods have "AF" in the name now, bit the other methods do not have AF?